lib/PSR/HTTP/Server.hs

```diff
@@ -33,11 +33,11 @@ server Storage{..} = siteH
             }
 
     eventsHandler filterParams' mName = do
-        -- The capture parameter `name` has a higher priority over the query param
-        let nameFilterParameter = maybe (_filterQueryParam_name filterParams') Just mName
-        let filterParams = filterParams'{_filterQueryParam_name = nameFilterParameter}
-        _events <- liftIO $ getEvents filterParams
-        pure []
+        -- The capture parameter `name_or_script_hash` has a higher priority over the query param
+        let nameOrScriptHashFilterParameter = maybe (_eventFilterParam_name_or_script_hash filterParams') Just mName
+        let filterParams = filterParams' {_eventFilterParam_name_or_script_hash = nameOrScriptHashFilterParameter}
+        events <- liftIO $ getEvents filterParams
+        pure $ events <&> \e -> API.Event e.eventType e.blockHeader e.createdAt
```

Comment:
No `<|>` but we're pulling out `<&>`? 😂  (I'm ok with using both though)

I wish `<&>` was exported by Prelude, I often find it much more readable than `<$>`.